masterlist.yaml

```diff
@@ -6769,6 +6769,47 @@ plugins:
       # version: 1.06a
       - crc: 0x20AA5984
         util: 'SSEEdit v3.2.2'
+  - name: 'Immersive Movement.esp'
```

Comment:
Overall placement generally doesn't make a difference, but I'd prefer if this were placed somewhere more appropriate. If you're unsure, then just place new entries at the end of the masterlist.

Reply:
Moved to Gameplay section.